src/installation/installation.md

```diff
@@ -1,32 +1,109 @@
-# Installation
+# Installation of Epinio
 
-## System requirements
+## Introduction
+Epinio is installed from a single Helm chart and, by default, it also installs [`Kubed`](#kubed) and [`MinIO`](#s3-storage) in your Kubernetes cluster.
```

Comment:
Probably it's worth saying that it will also install by default an internal container registry, and that you can disable it if you want. :slightly_smiling_face:

Reply:
Added the reference to the container registry.